services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -215,14 +216,19 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
   };
 
   const getDrawerTitle = () => {
+    let addOrEdit = "";
     let name =
       selectedEntity?.name ||
       selectedEntity?.custom_analysis_method_name ||
       "Add other";
     if (isReportingOnStandards && report) {
+      addOrEdit = selectedEntity ? "Edit " : "Add ";
```

Comment:
nit: I don't think you need the spaces here (after the words 'edit' and 'add'), I think the utility handles it for you 🤔